src/utils/conversion.ts

```diff
@@ -976,6 +977,19 @@ export function transactionPermissionsToExtrinsicPermissions(
   );
 }
 
+/**
+ * @hidden
+ */
+export function transactionPermissionsToExtrinsicPermissions(
```

Comment:
Is this used anywhere now? If no, should we remove this?

Reply:
I added this in to be called from 

`permissionsToMeshPermissions`.

The error would come from the inviteAccount permissions calling createType where it was a `PolymeshPrimitivesSubsetSubsetRestrictionPalletPermissions` type. By giving it an `ExtrinsicPermissions` type instead it seems to resolve it.